agent/github_tools.py

```
@@ -455,20 +472,25 @@ async def create_branch(self, branch_name: str, base_branch: str = "main") -> di
+        # in .git/config is a security risk.
         await self._run("git", "remote", "set-url", "origin", self.clone_url)
-        rc, out, err = await self._run(
-            "git", "push", "--set-upstream", "origin", branch
-        )
-        # Always clear the token from remote URL
-        await self._run("git", "remote", "set-url", "origin",
-                        f"https://github.com/{self.owner}/{self.repo}.git")
-        if rc != 0:
-            raise RuntimeError(f"git push failed: {err}")
+        try:
+            rc, out, err = await self._run(
```

Ruff (0.15.15)

[warning] Unpacked variable `out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)